### DISCUSSION TITLE
Have a middle_name field on Name.schema.json

### DISCUSSION THREAD

**Discussion by jacobyavis** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#237] [Ideas]
Should be optional of course, but many systems have this stored. Is there a reason we excluded it from OCF?

**Comment by JSv4** [comment]
Would you be opposed to ditching first name and last name and just using the legal name instead, @jacobyavis? Another angle her worth considering is the first and last name fields don't capture naming conventions in many parts of the world. Maybe we don't even need the type?

**Reply by pjohnmeyer** [reply]
This was my expressed preference as well. There is no (reasonable) structured way to properly represent all names and naming conventions. If we replace first_name and last_name with (still optional) `given_name` and `family_name` that gets us closer, but then what about name suffixes (Jr., Sr., III)? What if the individual has multiple given / family names -- is that a single string or an array of strings? How does each platform separate?

[w3.org article on form design for names](https://www.w3.org/International/questions/qa-personal-names#singlefield)

Based on that article, and on my own experiences dealing with certain folks, I would be tempted to suggest, if anything, that we have an optional "display_name" / "chosen_name" for UIs, emails, snail mails, etc. that is similarly a single string.
